src/draw/lv_draw.h

```diff
@@ -116,8 +116,10 @@ typedef struct _lv_draw_unit_t {
      * A draw task should be assign only if the draw unit can draw it too
      * @param draw_unit     pointer to the draw unit
      * @param layer         pointer to a layer on which the draw task should be drawn
-     * @return              >=0:    The number of taken draw task
-     *                      -1:     There where no available draw tasks at all.
+     * @return              >=0:    The number of taken draw task:
+     *                                  0 means the task has not yet been completed.
+     *                                  1 means a new task has been accepted.
+     *                      -1:     There are no available draw tasks at all.
```

Comment:
```suggestion
     *                      -1:     The draw unit wanted to work on on a task but couldn't do that due to some errors (e.g. out of memory). 
     *                              It signals that LVGL should call the dispatcher later again to let draw unit try to start the rendering again.  
```

Reply:
I believe the return code of any unit dispatcher shall be like this:
0   - unit is busy with some other task or unit does not support any new task from the list
1   - new task has been taken
-1 - out of memory during layer allocation or there are no more tasks in the list.

Reply:
>  0 - unit is busy with other task or unit does not support any new task from the list

Clear: in both cases a dispatch request will be sent when the current draw task or an other draw task in an other unit is ready, so this unit can try render a task. I would add this longer description in the comments because these parts are really complicated.

>  1 - new task has been taken

Also clear. Maybe we can return the number of taken tasks, but I don't know for what we can use that information

> -1 - out of memory during layer allocation or there are no more tasks in the list.

1. Out of memory: it's fine, but let's describe that LVGL will try to dispatach later
2. No more tasks: it's a different case as no more dispatching is required. We can see it from `layer->draw_task_head == NULL`

Reply:
I have updated description for -1. Please check it.